mirrord/auth/src/credential_store.rs

```diff
@@ -33,11 +34,12 @@ static CREDENTIALS_PATH: LazyLock<PathBuf> = LazyLock::new(|| CREDENTIALS_DIR.jo
 /// Container that is responsible for creating/loading `Credentials`
 #[derive(Default, Debug, Serialize, Deserialize)]
 pub struct CredentialStore {
-    /// User-specified CN to be used in all new certificates in this store. (Defaults to hostname)
-    common_name: Option<String>,
     /// Credentials for operator
-    /// Can be linked to several diffrent operator licenses via diffrent keys
+    /// Can be linked to several different operator licenses via different keys.
+    #[serde(default)]
     credentials: HashMap<String, Credentials>,
+    #[serde(default)]
+    signing_keys: HashMap<String, KeyPair>,
```

Comment:
>  Therefore, we can use the public key to identify users across renewals.

This is what's going on here, right?

Reply:
Yeah, whenever we create/retrieve user's `Credentials`, we associate the found key pair with operator's subscription id. Then, the operator's license is renewed - its fingerprint changes and we don't have any matching `Credentials`. But the subscription id does not change, so we look up the `signing_keys` mapping to find the key pair we used previously.
Also, subscription id is passed around as an `Option` to make the CLI backwards compatible

Reply:
Alright, just add this as docs then.